.github/workflows/integrationChartBump.yaml

```
@@ -25,21 +25,25 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
+          path: datarepo-helm
+          token: ${{ secrets.BROADBOT_TOKEN }}
 
       - name: 'Checkout ${{ matrix.repo }} repo'
         uses: actions/checkout@v2
         with:
           repository: 'databiosphere/${{ matrix.repo }}'
-          token: ${{ secrets.CR_TOKEN }}
+          token: ${{ secrets.BROADBOT_TOKEN }}
           path: ${{ matrix.repo }}
-          persist-credentials: false
 
       - name: "Find latest version chart version"
         id: chartversion
         run: |
           for i in $(echo ${{ env.charts }})
           do
-            echo "::set-output name=$i-version::$(docker run --rm -v ${PWD}:/workdir mikefarah/yq yq read charts/$i/Chart.yaml version)"
+            val=$(docker run --rm -v ${PWD}:/workdir mikefarah/yq:3 yq read datarepo-helm/charts/$i/Chart.yaml 'version')
+            echo "New $i version $val"
+            echo "::set-output name=$i-version::$val"
```

```suggestion
            CHART_VERSION=$(docker run --rm -v ${PWD}:/workdir mikefarah/yq:3 yq read datarepo-helm/charts/$i/Chart.yaml 'version')
            echo "New $i version $CHART_VERSION"
            echo "::set-output name=$i-version::$CHART_VERSION"
```

Perhaps uppercase and set a more descriptive name?